docs/coding.rst

```diff
@@ -0,0 +1,100 @@
+Coding Conventions
+==================
+
+Code Formatting
+---------------
+
+.. _`yapf`: https://github.com/google/yapf
+
+We use `yapf`_ to format all of the code in DeepChem.  Although it sometimes
+produces slightly awkward formatting, it does have two major benefits.  First,
+it ensures complete consistency throughout the entire codebase.  And second, it
+avoids disagreements about how a piece of code should be formatted.
+
+Whenever you modify a file, run :code:`yapf` on it to reformat it before
+checking it in.
+
+.. code-block:: bash
+
+  yapf -i <modified file>
+
+Yapf is run on every pull request to make sure the formatting is correct, so if
```

Comment:
Could you add a line about making sure to use the same version of `yapf` as DeepChem is currently using? We're on 0.22, and there seem to be big differences between the latest 0.30 and 0.22

Reply:
Good idea!  I added it.